src/vehicle.cpp

```diff
@@ -1502,27 +1521,37 @@ bool vehicle::start_engine( const int e )
         }
     }
 
-    if( einfo.fuel_type == fuel_type_gasoline || einfo.fuel_type == fuel_type_diesel ) {
-        // Small engines can be started without a battery (pull start or kick start)
-        if( engine_power >= 50 ) {
-            const int penalty = ((engine_power * dmg) / 2) + ((engine_power * cold_factor) / 5);
-            if( discharge_battery( (engine_power + penalty) / 10, true ) != 0 ) {
-                add_msg( _("The %s makes a rapid clicking sound."), einfo.name.c_str() );
-                return false;
-            }
-        }
+    // Immobilisers need removing before the vehicle can be started
+    if( eng.faults().count( fault_immobiliser ) ) {
+        add_msg( _( "The %s makes a long beeping sound." ), eng.name().c_str() );
+        return false;
+    }
 
-        // Damaged engines have a chance of failing to start
-        if( x_in_y( dmg * 100, 120 - (20 * cold_factor) ) ) {
-            if( one_in( 2 ) ) {
-                add_msg( _("The %s makes a deep clunking sound."), einfo.name.c_str() );
-            } else {
-                add_msg( _("The %s makes a terrible clanking sound."), einfo.name.c_str() );
-            }
+    // Engine with starter motors can fail on both battery and starter motor
+    if( eng.faults_potential().count( fault_starter ) ) {
+        if( eng.faults().count( fault_starter ) ) {
+            add_msg( _( "The %s makes a single clicking sound." ), eng.name().c_str() );
+            return false;
+        }
+        const int penalty = ( engine_power * dmg / 2 ) + ( engine_power * cold_factor / 5 );
+        if( discharge_battery( ( engine_power + penalty ) / 10, true ) != 0 ) {
+            add_msg( _( "The %s makes a rapid clicking sound." ), eng.name().c_str() );
             return false;
         }
```

Comment:
You still can providing the engine doesn't define `fault_engine_starter`. An example of this would be `1cyl_combustion_small`. In part 2 I want to overhaul the engine weights, volumes and descriptions and make pull/kick-starting more of an explicit feature.